Cargo.toml

```diff
@@ -6,11 +6,8 @@ edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
-[lib]
-name = "core"
-path = "src/core/lib.rs"
-
 [dependencies]
+hcor = { path = "../hcor" }
```

Comment:
This should depend on the git repository at hackagotchi/hcor, not rely on the user having their git repositories set up in a particular way.